Tests/Engine/InvokeScriptAnalyzer.tests.ps1

```diff
@@ -497,3 +497,10 @@ Describe "Test -Fix Switch" {
         $actualScriptContentAfterFix | Should Be $expectedScriptContentAfterFix
     }
 }
+
+Describe "Test -EnableExit Switch" {
+    It "Returns exit code equivalent to number of warnings" {
+        $process = Start-Process powershell -ArgumentList '-WindowStyle Hidden -Command Import-Module PSScriptAnalyzer; Invoke-ScriptAnalyzer -ScriptDefinition gci -EnableExit' -PassThru -Wait
+        $process.ExitCode | Should Be 1
+    }
```

Comment:
What you have is fine, but would this also work?
```
powershell -command 'import-module psscriptanalyzer; invoke-scriptanalyzer -enableexit -scriptdef gci'
$LASTEXITCODE | Should be 1
```

Eventually, I would like to see these be portable on other platforms, and the `-WindowStyle` parameter isn't portable. Eventually, we also need to calculate the name of `powershell` (since it can be powershell or pwsh, but that can be ignored for now)

Reply:
Yes, that is possible. I have changed it to that.